
simulation_step: 0.01
random_seed: 112392
environment: "environments/empty.yaml"

plant:
  name: "fg_ltv_sde"
  path: "fg_ltv_sde"
  
  start_state: [0,0,0,0]

  state_space:
    lower_bound: [-100, -100, -10, -10]
    upper_bound: [100, 100, 10, 10]
  
  control_space:
    lower_bound: [-10,-10]
    upper_bound: [ 10, 10]


  vis_body: "body"